src/components/avatar/Avatar.scss

```diff
@@ -81,6 +97,20 @@
                 left: calc(100% - 10px);
             }
         }
+
+        &.avatar--iconBadge {
+            line-height: 0; // .badgeable-container has `line-height: 1 which evaluates to 13px which is too much for small badge
+
+            @include Badgeable-bottomRightBadge {
```

Comment:
Is this mixin call needed? It should be inherited by the styles above in the base class?